ecosystem/node-checker/src/evaluators/metrics/state_sync/version.rs

```diff
@@ -227,8 +229,8 @@ mod test {
         };
 
         let state_sync_metrics_evaluator =
-            StateSyncVersionEvaluator::new(StateSyncVersionEvaluatorArgs {
-                version_delta_tolerance: 1000,
+            StateSyncVersionMetricsEvaluator::new(StateSyncVersionMetricsEvaluatorArgs {
+                metrics_version_delta_tolerance: 1000,
```

Comment:
and this is even smaller :)

why is this different from the default?

Reply:
Just a unit test, given the numbers are artificial I didn't think it mattered too much. I can update them to be more realistic.